### PR TITLE
Add images and some minor UG fix

### PR DESCRIPTION
**Pull request by muhammad-faruq**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -245,7 +255,21 @@ The proposed theme mechanism is simply done by completely altering the base css
...
-### \[Proposed\] Genre adding feature
+**Current Implementation**
+
+CSS files have been optimized for theme creation. Devs are only needed to edit the list of colors at the top of the CSS file
+and the source path for the background image.
+* Pros: More advanced users may be able to create their own CSS files by setting the color and changing the source of
+the background image.
+* Cons: There is a limited choice of color blocking. The bulk of the theming is made from a specified color shading.
+
+**Proposed Implementation**
```
Comment (auhc99) [inline docs/DeveloperGuide.md:266]: why is there a proposed implementation when you are describing a current feature in our app?

**Review by auhc99 — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (luodan01) [reply docs/DeveloperGuide.md:266]: @auhc99 I think this is allowed, plus the feature of allowing users to make their own themes feels like a new feature
image

Comment (muhammad-faruq) [reply docs/DeveloperGuide.md:266]: So this all good ye? I wanted to add it as possible improvements to the project for future devs

Comment (auhc99) [reply docs/DeveloperGuide.md:266]: Then shouldn't this be under another header called [Proposed] Themes Adding Feature?

Comment (muhammad-faruq) [reply docs/DeveloperGuide.md:266]: fair.. will make the changes then

**Pull request merged**